tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -1671,6 +1671,32 @@ def forward_xp(self, inputs, xp):
         return xp.sigmoid(inputs[0]),
 
 
+@op_utils.op_test(['native:0', 'cuda:0'])
+@chainer.testing.parameterize(*(
+    # Special shapes
+    chainer.testing.product({
+        'shape': [(), (0,), (1,), (2, 0, 3), (1, 1, 1), (2, 3)],
+        'in_dtypes,out_dtype': _in_out_float_dtypes_math_functions,
```

Comment:
Why not testing also with integer dtypes?
```suggestion
        'in_dtypes,out_dtype': _in_out_dtypes_math_functions,
```